# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gjs/hotfix-no-tests.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gjs-1.80.2/meson.build.vanilla	2024-04-23 15:31:55.896000000 +0200
+++ gjs-1.80.2/meson.build	2024-04-23 15:30:37.205000000 +0200
@@ -684,6 +684,7 @@
         'resource:///org/gnome/gjs')
 endif
 
+if false
 ### Tests and test setups ######################################################
 
 subdir('installed-tests')
@@ -765,6 +766,8 @@
     warning('Not using DBus, not all tests will be run.')
 endif
 
+endif
+
 ### Summarize options ##########################################################
 
 prefix = get_option('prefix')
